app/data/AtomDataStore.scala

```diff
@@ -0,0 +1,17 @@
+package data
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.gu.atom.data.PreviewDynamoDataStore
+import com.gu.contentatom.thrift._
+import com.gu.contentatom.thrift.atom.media.MediaAtom
+import com.gu.scanamo.scrooge.ScroogeDynamoFormat._
+import com.gu.scanamo.DynamoFormat
+import DynamoFormat._
+import cats.syntax.either._
+
+class PreviewAtomDataStore(dynamo: AmazonDynamoDBClient, tableName: String) {
```

Comment:
This could possibly become `AtomDataStores`, with a `val previewStore` and `val publishedStore` inside it